doc/classes/Camera2D.xml

```diff
@@ -5,7 +5,8 @@
 	</brief_description>
 	<description>
 		Camera node for 2D scenes. It forces the screen (current layer) to scroll following this node. This makes it easier (and faster) to program scrollable scenes than manually changing the position of [CanvasItem]-based nodes.
-		This node is intended to be a simple helper to get things going quickly and it may happen that more functionality is desired to change how the camera works. To make your own custom camera node, simply inherit from [Node2D] and change the transform of the canvas by calling get_viewport().set_canvas_transform(m) in [Viewport].
+		This node is intended to be a simple helper to get things going quickly and it may happen that more functionality is desired to change how the camera works. To make your own custom camera node, inherit from [Node2D] and change the transform of the canvas by calling [method Viewport.set_canvas_transform] in [Viewport] (you can obtain the current [Viewport] by using [method Node.get_viewport].
```

Comment:
```suggestion
		This node is intended to be a simple helper to get things going quickly and it may happen that more functionality is desired to change how the camera works. To make your own custom camera node, inherit from [Node2D] and change the transform of the canvas by calling [method Viewport.set_canvas_transform] in [Viewport] (you can obtain the current [Viewport] by using [method Node.get_viewport]).
```